torchvision/utils.py

```diff
@@ -115,7 +115,7 @@ def norm_range(t, value_range):
 def save_image(
     tensor: Union[torch.Tensor, List[torch.Tensor]],
     fp: Union[Text, pathlib.Path, BinaryIO],
-    normalize: bool = False,
+    normalize: Union[bool, str] = 'deprecated',
```

Comment:
This one throw me off as well. I think the normalize should not be defined here and it was meant instead to be captured in the `**kwargs` and passed tot he `make_grid` method. See here:
https://github.com/pytorch/vision/blob/0e6dae17fcb561af15fb320ba8fe04b34cd36cc7/torchvision/utils.py#L14-L18

So this is probably more of a bug than a documentation problem.

Reply:
ha yes good point, this was probably intended for `make_grid`.

Since the arguments are not keyword-only, removing `normalize` from the signature could potentially break existing code. It might be preferable to just leave `normalize` in `save_image` and properly pass it down in the call to `save_image`.

Reply:
Removing the `normalize` from the signature wont break any code but rather will fix the bug. By having the normalize as a separate variable, we cause it to not be included in the `**kwargs` and subsequently passed to the `make_grid`. This is a bug and it's worth fixing.

Reply:
> Removing the normalize from the signature wont break any code

I believe it can break code that uses pure positional parameter passing e.g.:

```py
save_image(tensor, fp, False)
```

If we remove `normalize`, `False` is now mapped to the `format` parameter which will at best lead to an error, at worst a silent bug.

I would agree it's an unlikely scenario but it can exist

Reply:
@NicolasHug Here are are my 2 cents on this:

1. The use of `save_image(tensor, fp, False)` (positional vs named) does not seem like a very likely scenario. We should note that the specific parameter is broken and has no effect at the moment.
2. I think we should opt for cleaning up and simplifying our API and resolving the bug.

Reply:
OK, I'll let `normalize` be part of `kwargs` and we can advertise this as a bug-fix then. Thanks for the input

Reply:
For future reference, the change discussed here was moved on a separate PR at #3324.